### DISCUSSION TITLE
icon/tags for users

### DISCUSSION THREAD

**Discussion by Midnight-NOW** [vrcx-team/VRCX#632] [Ideas]
Add a way to add tags that with show up on the overlays works for anyone, don't need to be friends. so if you know that you have a Toxic player join shows ( [Toxic] already exists, Has joined ) so you have a bit of info as they join or a heart for loved ones etc.

**Comment by Natsumi-sama** [comment]
It *was* planned but not looking likely with the current length of my todo list.
